ansible/roles/sidekiq/files/sidekiq-service

```diff
@@ -0,0 +1,2 @@
+deploy ALL=(root) NOPASSWD: /usr/sbin/service
+deploy ALL=(root) NOPASSWD: /usr/bin/monit
```

Comment:
the service sudo should be in common and the monit one should be in monit, right?